uber/static/theme/prereg.css

```diff
@@ -64,3 +64,10 @@ body {
 .panel.panel-default {
     padding-top: 15px;
 }
```

Comment:
I'm terrible about color schemes and such, so if Nick or whoever wants the notice to be a different color or have a different background or something, just edit this here.